variation/validators/genomic_deletion_range.py

```diff
@@ -0,0 +1,249 @@
+            `False` otherwise.
+        :return: Sequence Interval and GRCh38 data if normalize endpoint
+            is being used
+        """
+        ival, grch38, start1, start2, end1, end2 = None, None, None, None, None, None  # noqa: E501
+        # (#_#)_(#_#)
+        if is_norm:
+            t, start1, start2, end1, end2, grch38 = self.get_grch38_pos_ac(
+                t, s.start_pos1_del, s.start_pos2_del, pos3=s.end_pos1_del,
+                pos4=s.end_pos2_del
+            )
+        else:
+            start1 = s.start_pos1_del
+            start2 = s.start_pos2_del
+            end1 = s.end_pos1_del
+            end2 = s.end_pos2_del
+
+        gene = gene_tokens[0].token if gene_tokens else None
+        self.validate_gene_or_accession_pos(
+            t, [start1, start2, end1, end2], errors, gene=gene)
+
+        if not errors and start1 and start2 and end1 and end2:
+            ival = self._get_ival_certain_range(
+                start1, start2, end1, end2
+            )
+
+        return ival, grch38
+
+    def get_gene_tokens(
+            self, classification: Classification) -> List[GeneMatchToken]:
+        """Return gene tokens for a classification.
+
+        :param Classification classification: The classification for tokens
+        :return: A list of Gene Match Tokens in the classification
+        """
+        return self.get_gene_symbol_tokens(classification)
+
+    def variation_name(self) -> str:
+        """Return the variation name."""
+        return 'genomic deletion range'
+
+    def is_token_instance(self, t: Token) -> bool:
+        """Check that token is Genomic Deletion Range.
+
+        :param Token t: Classification token
+        """
+        return t.token_type == 'GenomicDeletionRange'
+
+    def validates_classification_type(
+            self,
+            classification_type: ClassificationType) -> bool:
+        """Return whether or not the classification type is
+        Genomic Deletion Range.
+
+        :param ClassificationType classification_type: Classification type
+        :return: `True` if classification type matches, `False` otherwise
+        """
+        return classification_type == \
+            ClassificationType.GENOMIC_DELETION_RANGE
+
+    def human_description(self, transcript,
+                          token: GenomicDeletionRangeToken) -> str:
+        """Return a human description of the identified variation."""
+        descr = f"A Genomic Deletion from" \
+                f" ({token.start_pos1_del}_{token.start_pos2_del}) to " \
+                f"({token.end_pos1_del}_{token.end_pos2_del}) on {transcript}"
+        return descr
+
+    def concise_description(self, transcript, token):
+        """Return a concise description of the identified variation."""
+        return f"{transcript}:g.({token.start_pos1_del}_" \
+               f"{token.start_pos2_del})_({token.end_pos1_del}_" \
+               f"{token.end_pos2_del})"
```

Comment:
types

Reply:
I'm going to be getting rid of these in #161 , but can still address this

Reply:
Oh if they're getting trashed then disregard